cmd/testsite/golden/basic-test1/test-library/R6/INDEX

```diff
@@ -0,0 +1,3 @@
+R6Class                 Create an R6 reference object generator
```

Comment:
Do we really care about checking in all information from the package, especially the INDEX and binary RDS files? Wouldn't we just want to keep around the folder/DESCRIPTION file?